test/e2e/pom.xml

```diff
@@ -44,6 +44,8 @@
     </modules>
 
     <properties>
+        <sw.version></sw.version> <!-- Please manually set the SkyWalking version here to build in the IDE, but make sure not to check it into the code base / git -->
```

Comment:
```suggestion
        <sw.version></sw.version> <!-- Please manually set the SkyWalking version here to build in the IDE, such as -Dsw.version=x.y.z, but make sure not to check it into the code base / git -->
```
Is this correct?

Reply:
`-Dsw.version=x.y.z` should be in command line, if in IDE, should just put x.y.z in <sw.version></sw.version> like <sw.version>x.y.z</sw.version>

Reply:
Or VM option of the IDE, right? 

I still feel, we should add a check step in the CI, before we run maven.

Reply:
And remember to update the e2e doc.

Reply:
> Or VM option of the IDE, right?
> 
> I still feel, we should add a check step in the CI, before we run maven.

Check what? The property is already empty.

Reply:
Use shell to read this property before run `./mvnw`, if it is not empty, we should shut down the e2e directly with warning.